pio/st7789_lcd/st7789_lcd.pio

```diff
@@ -33,6 +33,11 @@ static inline void st7789_lcd_program_init(PIO pio, uint sm, uint offset, uint d
     sm_config_set_fifo_join(&c, PIO_FIFO_JOIN_TX);
     sm_config_set_clkdiv(&c, clk_div);
     sm_config_set_out_shift(&c, false, true, 8);
+	
+    // The pin muxes can be configured to invert the output (among other things
```

Comment:
It appears to me that the closing paren must've been omitted:
```diff
-// The pin muxes can be configured to invert the output (among other things
+// The pin muxes can be configured to invert the output (among other things),
```

Reply:
Thanks for your help! I have no merging powers vested in me :) From my experience, the repo maintainers approved and merged after quite some time, before releasing the new SDK version.